### ISSUE TITLE
*help* changing gradients  value through Masking the gradients with a matrix whose shape same with var but non-compatible

### ISSUE THREAD

**Issue by PlayDeep** [tensorflow/tensorflow#27509]
<em>Please make sure that this is a documentation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:doc_template</em>

**System information**
- TensorFlow version: tf 1.8
- Doc Link:https://tensorflow.google.cn/versions/r1.8/api_docs/python/tf/train/AdamOptimizer?hl=zh-cn#compute_gradients

> This is the first part of minimize(). It returns a list of (gradient, variable) pairs where "gradient" is the gradient for "variable". Note that "gradient" can be a Tensor, an IndexedSlices, or None if there is no gradient for the given variable.

**Describe the documentation issue**
I have a corresponding mask matrix which is obtained with a trained model, and using the code as following. 
`grad = tf.multiply(grad.values, tf.convert_to_tensor(tf.abs(gradient_mask[name])), name='%s_pruning_mask'%name)`   it is report a error of compatible because this gradient has  a shape(17374, 512) and matrix shape(17353, 512). In Truth, I have to found the gradient shape is correspond other tensors but found nothing . the mask matrix is target embeddings weights shape. So what's wrong with this code or documents, why? please and Thank you.

**We welcome contributions by users. Will you be able to update submit a PR (use the [doc style guide](https://www.tensorflow.org/community/documentation)) to fix the doc Issue?**

**Comment by ymodak** [comment]
Apologies for the delay in response. If you think it's a bug, please provide a minimal code snippet to reproduce the issue.
If you think we need to update our documents; please can you elaborate on what changes are you looking to be made. Thanks!

**Comment by ymodak** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
